components/BottomSheet/src/MDCBottomSheetTransitionController.m

```diff
@@ -194,4 +196,8 @@ - (UIAccessibilityTraits)scrimAccessibilityTraits {
   return _scrimAccessibilityTraits;
 }
 
+- (void)setElevation:(MDCShadowElevation)elevation {
```

Comment:
Can this be removed? It looks like the default behavior for autosynthesized setters.

Reply:
Done